tmva/tmvagui/inc/TMVA/mvaeffs.h

```diff
@@ -15,7 +15,8 @@
 
 namespace TMVA{
 
-   void mvaeffs(TString dataset, TString fin = "TMVA.root", 
+   void mvaeffs(TString dataset, TString fin = "TMVA.root",
+                Float_t nSingal = 1000, Float_t nBackground = 1000,
```

Comment:
Why do you put 1000 here? I don't understand it :thinking:

Reply:
ah, and there's a typo in `nSingal`!

Reply:
1000 are default values previously hard-coded in the code (line 533 in mvaeffs.cxx) 
```
StatDialogMVAEffs* gGui = new StatDialogMVAEffs(dataset, graphicsClient->GetRoot(), 1000, 1000);
```

Thank you for finding the typo. 

I will update now this PR with the fix for avoiding #6483

Reply:
Ah I see! Thanks :)